engine/launcher/src/main/scala/org/enso/launcher/cli/Main.scala

```diff
@@ -319,18 +320,30 @@ object Main {
     }
 
   private def uninstallEngineCommand: Subcommand[Config => Unit] =
-    Subcommand("engine") {
+    Subcommand(
+      "engine",
+      "Uninstalls a provided engine version. If a runtime is not used by any " +
+      "engines anymore, it is also removed."
+    ) {
       val version = Opts.positionalArgument[SemVer]("VERSION")
       version map { version => (config: Config) =>
         Launcher(config).uninstallEngine(version)
       }
     }
 
   private def uninstallDistributionCommand: Subcommand[Config => Unit] =
-    Subcommand("distribution") {
-      Opts.pure(()) map { (_: Unit) => (_: Config) =>
-        Logger.error("Not implemented yet.")
-        sys.exit(1)
+    Subcommand(
+      "distribution",
+      "Uninstalls the whole Enso distribution and all components managed by " +
+      "it. If `auto-confirm` is set, does not attempt to remove the " +
+      "ENSO_DATA_DIRECTORY and ENSO_CONFIG_DIRECTORY if they contain any " +
+      "unexpected files."
```

Comment:
```suggestion
      "Uninstalls whole Enso distribution and all components managed by " +
      "it. If `auto-confirm` is set, it will not attempt to remove the " +
      "ENSO_DATA_DIRECTORY and ENSO_CONFIG_DIRECTORY if they contain any " +
      "unexpected files."
```